src/gql/queries/user-subscriptions.graphql

```diff
@@ -0,0 +1,54 @@
+query UserSubscriptions {
+  user {
+    subscriptions {
+      id
+      ownerType
+      regexSelectors {
+        data
+        type
+      }
+      resourceType
+      selectors {
+        data
+        type
+      }
+      subscriber {
+        subscriber {
+          emailSubscriber
+          githubCheckSubscriber {
+            owner
+            ref
+            repo
+          }
+          githubPRSubscriber {
+            owner
+            prNumber
+            ref
+            repo
+          }
+          jiraCommentSubscriber
+          jiraIssueSubscriber {
+            issueType
+            project
+          }
+          slackSubscriber
+          webhookSubscriber {
+            headers {
+              key
+              value
+            }
+            minDelayMs
+            retries
+            secret
+            timeoutMs
+            url
+          }
+        }
```

Comment:
I wonder if we need to request all of these subscribers for user subscriptions; I had an impression that we only offered `emailSubscriber`, `jiraCommentSubscriber`, and `slackSubscriber` at a user level